src/store/sagas/tezos/blokchain.ts

```diff
@@ -10,7 +10,7 @@ import apiConfig from "./api-config";
 import * as blokchainActions from "../../actions/tezos/blokchain";
 import * as loaderActions from "../../actions/loader";
 
-const initialFetchAmount = 100000;
+const initialFetchAmount = 10000;
```

Comment:
Moze zmien to na 40k, bo tyle z ethereum pobiera i wszystko bedzie szybciej dzialac.

Reply:
może być i 40k, ale czemu ma być szybciej jak będzie więcej? (wgl to ja to niechcący zacommitowałem 😅)